src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -101,32 +102,38 @@ def get_settings_wdg(my):
         from pyasm.widget import SelectWdg
         search_type_obj = SearchType.get(my.search_type)
         base_type = search_type_obj.get_base_key()
-        search = Search("sthpw/pipeline")
-        search.add_filter("search_type", base_type)
-        pipelines = search.get_sobjects()
-        if pipelines:
-            pipeline = pipelines[0]
 
-            process_names = pipeline.get_process_names()
-            if process_names:
-                title_wdg = DivWdg()
-                div.add(title_wdg)
-                title_wdg.add("Process")
-                title_wdg.add_style("margin-top: 20px")
-                title_wdg.add_style("font-size: 16px")
+        process_names = []
+        pipeline_search = Search("sthpw/pipeline")
+        pipeline_search.add_filter("search_type", base_type)
+        pipelines = pipeline_search.get_sobjects()
+        for pipeline in pipelines:
+            process_names.extend(pipeline.get_process_names())
+  
+        selected_process = my.kwargs.get("process")
+        if selected_process:
+            process_names.append(selected_process)
+        process_names = list(set(process_names))
+
+        title_wdg = DivWdg()
+        div.add(title_wdg)
+        title_wdg.add("Process")
+        title_wdg.add_style("margin-top: 20px")
+        title_wdg.add_style("font-size: 16px")
 
-                desc_wdg = DivWdg("Select which process to ingest these files into")
-                div.add(desc_wdg)
+        desc_wdg = DivWdg("Select which process to ingest these files into")
```

Comment:
I think we are going with a shorter select label and optionally longer selection item  in general.  Let's shorten it to Process: "Select Ingest Process".. where Select Ingest Process is the first item with a value = ''